targets/common

```diff
@@ -89,6 +89,11 @@ elif [ -z "$TARGETNOINSTALL" -o -n "$RESTOREHOSTBIN" ]; then
         $SIMULATE cp -fP "$src" "$CHROOT/etc/crouton/"
         [ ! -h "$src" ] && $SIMULATE chmod 644 "$CHROOT/etc/crouton/$f"
     done
+
+    # Copy all the source files to chroot
+    $SIMULATE rm -rf "$CHROOT/usr/src/crouton"
```

Comment:
if the first thing you do is rm -rf, then it sounds like you should be creating a temporary directory for the source

Reply:
Probably yes, I just didn't feel like passing yet another variable for prepare.sh for that... (unless I forgot something about how we can pass variables from outside to inside chroot).

Reply:
Ah, okay...